cfme/tests/cloud/test_targeted_refresh.py

```diff
@@ -20,9 +53,34 @@ def test_ec2_targeted_refresh_instance():
             2. Instance RUNNING
             3. Instance STOPPED
             4. Instance UPDATE
-            5. Instance DELETE - or - Instance TERMINATE
+            5. Instance RUNNING
+            6. Instance DELETE - or - Instance TERMINATE
     """
-    pass
+    vms_collection = appliance.rest_api.collections.vms
+    flavors_collection = appliance.rest_api.collections.flavors
+    instance = create_vm
+
+    # running
+    wait_for_power_state(vms_collection, instance.mgmt.name, "on")
```

Comment:
Can you just use `instance.name`, here and below?

Reply:
Well, instance.name != instance.mgmt.name.
I would have to rework the test as it is failing with instance.name.

Reply:
Ah, I wasn't aware they were different for EC2. We can leave it as is if you'd prefer.